pykafka/rdkafka/_rd_kafkamodule.c

```diff
@@ -0,0 +1,448 @@
+#include <Python.h>
+#include <structseq.h>
+
+#include <errno.h>
+
+#include <librdkafka/rdkafka.h>
+
+
+/**
+ * Exception types
+ */
+
+static PyObject *ConsumerStoppedException;
+static PyObject *PyRdKafkaError;
+
+
+static void
```

Comment:
No, I agree, I actually never wrote in this style before. I sort of went with what I saw in a lot of https://github.com/python/cpython code, but I don't have any preference here.